### PR TITLE
(tlg1286.tlg001-tlg014; tlg016-tlg020) CTS conversion and EpiDoc work 

### LINKED ISSUE #2259: (tlg1286.tlg001.1st1K-grc1.xml- tlg1286.tlg020.1st1K-grc1.xml) new edition
[urn.cts.greekLit.tlg1286.tlg001.1st1K-grc1.xml.zip](https://github.com/OpenGreekAndLatin/First1KGreek/files/5105246/urn.cts.greekLit.tlg1286.tlg001.1st1K-grc1.xml.zip)
[urn.cts.greekLit.tlg1286.tlg002.1st1K-grc1.xml.zip](https://github.com/OpenGreekAndLatin/First1KGreek/files/5105247/urn.cts.greekLit.tlg1286.tlg002.1st1K-grc1.xml.zip)
[urn.cts.greekLit.tlg1286.tlg003.1st1K-grc1.xml.zip](https://github.com/OpenGreekAndLatin/First1KGreek/files/5105248/urn.cts.greekLit.tlg1286.tlg003.1st1K-grc1.xml.zip)
[urn.cts.greekLit.tlg1286.tlg004.1st1K-grc1.xml.zip](https://github.com/OpenGreekAndLatin/First1KGreek/files/5105249/urn.cts.greekLit.tlg1286.tlg004.1st1K-grc1.xml.zip)
[urn.cts.greekLit.tlg1286.tlg005.1st1K-grc1.xml.zip](https://github.com/OpenGreekAndLatin/First1KGreek/files/5105250/urn.cts.greekLit.tlg1286.tlg005.1st1K-grc1.xml.zip)
[urn.cts.greekLit.tlg1286.tlg006.1st1K-grc1.xml.zip](https://github.com/OpenGreekAndLatin/First1KGreek/files/5105251/urn.cts.greekLit.tlg1286.tlg006.1st1K-grc1.xml.zip)
[urn.cts.greekLit.tlg1286.tlg007.1st1K-grc1.xml.zip](https://github.com/OpenGreekAndLatin/First1KGreek/files/5105252/urn.cts.greekLit.tlg1286.tlg007.1st1K-grc1.xml.zip)
[urn.cts.greekLit.tlg1286.tlg008.1st1K-grc1.xml.zip](https://github.com/OpenGreekAndLatin/First1KGreek/files/5105253/urn.cts.greekLit.tlg1286.tlg008.1st1K-grc1.xml.zip)
[urn.cts.greekLit.tlg1286.tlg009.1st1K-grc1.xml.zip](https://github.com/OpenGreekAndLatin/First1KGreek/files/5105254/urn.cts.greekLit.tlg1286.tlg009.1st1K-grc1.xml.zip)
[urn.cts.greekLit.tlg1286.tlg010.1st1K-grc1.xml.zip](https://github.com/OpenGreekAndLatin/First1KGreek/files/5105255/urn.cts.greekLit.tlg1286.tlg010.1st1K-grc1.xml.zip)
[urn.cts.greekLit.tlg1286.tlg011.1st1K-grc1.xml.zip](https://github.com/OpenGreekAndLatin/First1KGreek/files/5105256/urn.cts.greekLit.tlg1286.tlg011.1st1K-grc1.xml.zip)
[urn.cts.greekLit.tlg1286.tlg012.1st1K-grc1.xml.zip](https://github.com/OpenGreekAndLatin/First1KGreek/files/5105257/urn.cts.greekLit.tlg1286.tlg012.1st1K-grc1.xml.zip)
[urn.cts.greekLit.tlg1286.tlg013.1st1K-grc1.xml.zip](https://github.com/OpenGreekAndLatin/First1KGreek/files/5105258/urn.cts.greekLit.tlg1286.tlg013.1st1K-grc1.xml.zip)
[urn.cts.greekLit.tlg1286.tlg014.1st1K-grc1.xml.zip](https://github.com/OpenGreekAndLatin/First1KGreek/files/5105259/urn.cts.greekLit.tlg1286.tlg014.1st1K-grc1.xml.zip)
[urn.cts.greekLit.tlg1286.tlg016.1st1K-grc1.xml.zip](https://github.com/OpenGreekAndLatin/First1KGreek/files/5105260/urn.cts.greekLit.tlg1286.tlg016.1st1K-grc1.xml.zip)
[urn.cts.greekLit.tlg1286.tlg017.1st1K-grc1.xml.zip](https://github.com/OpenGreekAndLatin/First1KGreek/files/5105261/urn.cts.greekLit.tlg1286.tlg017.1st1K-grc1.xml.zip)
[urn.cts.greekLit.tlg1286.tlg018.1st1K-grc1.xml.zip](https://github.com/OpenGreekAndLatin/First1KGreek/files/5105262/urn.cts.greekLit.tlg1286.tlg018.1st1K-grc1.xml.zip)
[urn.cts.greekLit.tlg1286.tlg019.1st1K-grc1.xml.zip](https://github.com/OpenGreekAndLatin/First1KGreek/files/5105263/urn.cts.greekLit.tlg1286.tlg019.1st1K-grc1.xml.zip)
[urn.cts.greekLit.tlg1286.tlg020.1st1K-grc1.xml.zip](https://github.com/OpenGreekAndLatin/First1KGreek/files/5105264/urn.cts.greekLit.tlg1286.tlg020.1st1K-grc1.xml.zip)

(The edition does not include tlg1286.tlg015.1st1K-grc1.xml)

### PR DESCRIPTION
**Pull request by charhoughton**
CTS and EpiDoc work on tlg1286.tlg001 through tlg1286.tlg020, except tlg1286.tlg015.
fix #2259

### CODE REVIEW

**Comment by ahanhardt** [comment]
fix #2259

**Comment by lcerrato** [comment]
For these works, the spreadsheet has a different workgroup and different work titles. 
It's preferable to follow the spreadsheet (which is the TLG), so the workgroup is not Hermetica, but Corpus Hermeticum, and the work titles in the metadata should follow those examples. 

I am verifying this presently.

For instance, Poimandres is the work title for tlg1286.tlg001.

Also, the book title is not **Hermetica Vol One** but as it reads on the page:
**Hermetica. The Ancient Greek and Latin Writings which Contain Religious or Philosophic Teachings Ascribed to Hermes Trismegistus, Vol. I.**

File: data/tlg1286/tlg001/tlg1286.tlg001.1st1K-grc1.xml [hunk 1]
```diff
@@ -0,0 +1,543 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+   <teiHeader xml:lang="eng">
+       <fileDesc>
+           <titleStmt>
+               <title xml:lang="grc">[Ἑρμοῦ τρισμεγίστου] Ποιμάνδρης</title>
+               <author>Hermes Trimegistus</author>
```
Comment (lcerrato) [inline data/tlg1286/tlg001/tlg1286.tlg001.1st1K-grc1.xml:8]: Probably should be Corpus Hermeticum, but if kept as is, there is an "s" missing in the name.

File: data/tlg1286/tlg001/tlg1286.tlg001.1st1K-grc1.xml [hunk 2]
```diff
@@ -0,0 +1,543 @@
...
+               <pubPlace/>
+               <availability>
+                   <licence target="https://creativecommons.org/licenses/by-sa/4.0/">Available under a Creative Commons Attribution-ShareAlike 4.0 International License</licence>
+               </availability>
+               <date when="2020-08-19"/>
+           </publicationStmt>
+           <sourceDesc>
+               <biblStruct>
+                   <monogr><!--Removed author, since there is more than one--><!--Short version of title; no period; lang attribute-->
+                       <title xml:lang="lat">Hermetica Vol One</title>
```
Comment (lcerrato) [inline data/tlg1286/tlg001/tlg1286.tlg001.1st1K-grc1.xml:42]: Preferred title is 
Hermetica. The Ancient Greek and Latin Writings which Contain Religious or Philosophic Teachings Ascribed to Hermes Trismegistus.

Then the volume is indicated later in this header, not here.

File: data/tlg1286/tlg001/tlg1286.tlg001.1st1K-grc1.xml [hunk 3]
```diff
@@ -0,0 +1,543 @@
...
+                       <editor>Walter Scott</editor>
+                       <imprint>
+                           <pubPlace>Oxford</pubPlace>
+                           <publisher>Oxford University Press</publisher>
+                           <date>1924</date>
+                       </imprint>
+                   </monogr>
```
Comment (lcerrato) [inline data/tlg1286/tlg001/tlg1286.tlg001.1st1K-grc1.xml:49]: This is where the volume is indicated (following the `</imprint>` tag
`<biblScope unit="volume">1</biblScope>`

File: data/tlg1286/tlg001/tlg1286.tlg001.1st1K-grc1.xml [hunk 4]
```diff
@@ -0,0 +1,543 @@
...
+           </refsDecl><p>This pointer pattern extracts sections.</p>
+       </encodingDesc>
+       <profileDesc>
+           <langUsage>
+               <language ident="grc">Greek</language>
+               <language ident="lat">Latin</language>
+           </langUsage>
+       </profileDesc>
+       <revisionDesc><!-- put your name in 'who' and this date in YYYY-MM-DD format in 'when' -->
+           <change when="2020-02-02" who="Jane Smith">initial markup of new file.</change>
```
Comment (lcerrato) [inline data/tlg1286/tlg001/tlg1286.tlg001.1st1K-grc1.xml:73]: Is this a placeholder tag?

**Review by lcerrato — CHANGES_REQUESTED** [review]

**Comment by lcerrato** [comment]
@charhoughton @ahanhardt 

Thanks for this!

Please hold off for now as I am trying to work this and make an example of the first file.

There are a few issues here with titles, group name, author name, etc. These need a thorough review, but I am waiting on feedback.
The book title definitely needs revision in all of the cts metadata as well as the files themselves, and we have a separate tag to indicate volume.
I also see a "Jane Smith" in the change log which I believe is a placeholder?
In general, let's try to clean up the indentation a bit as this helps with my review tremendously.

I will try to make changes as I go, but this is a big pull request, so this is where having lots of files in one pull request doesn't work well as I have a few points in each file.

On first look, these are failing on a few points. We have a misnamed metadata file (__cts__ .xml  has an extra space ), some misnamed files ( urn.cts.greekLit.tlg1286.tlg004.1st1K-grc1.xml, | urn.cts.greekLit.tlg1286.tlg008.1st1K-grc1.xml) etc. So I have go through this little by little. Also note that there are unacceptable sections in tlg1286.tlg020.1st1K-grc.xml
https://travis-ci.org/github/OpenGreekAndLatin/First1KGreek/builds/740994284

I'm still trying to figure out the best approach so don't make any local changes until I can get the naming confirmed.

File: data/tlg1286/tlg001/tlg1286.tlg001.1st1K-grc1.xml [hunk 5]
```diff
@@ -0,0 +1,543 @@
...
+                        μου τῶν σωματικῶν αἰσθήσεων, ⟨οὐ μέντοι⟩ καθάπερ 
+                        ⟨τ⟩οῖ⟨ς⟩ [[ὕπνῳ]] βεβαρημένοι⟨ς⟩ ἐκ κόρου τροφῆς ἢ ἐκ κόπου <l n="5"/> 
+                        σώματος, ἔδοξά τινα ὑπερμεγέθη μέτρῳ ἀπεριορίστῳ ⟨ἐν⟩τνγχάνοντα 
+                        καλεῖν μου τὸ ὄνομα, καὶ λέγεν [τα] μοι· 
+                        Τί βούλει ἀκοῦσαι καὶ θεάσασθαι, καὶ νοήσας μαθεῖν καὶ 
+                        γνῶναι; — φημὶ ἐγώ· 
+                        </p>
+                    </div>
+                <div type="textpart" subtype="section" n="2">
+                    <p rend="merge">
```
Comment (lcerrato) [inline data/tlg1286/tlg001/tlg1286.tlg001.1st1K-grc1.xml:94]: `<p rend="merge">` not used. Simple `<p>` suffices when indentation is not used.

**Review by lcerrato — APPROVED** [review]

**Comment by lcerrato** [comment]
@charhoughton @ahanhardt 
Thank you for all of the work on this batch. I know they were challenging files!

**Pull request merged**